src/documentation/pages/components/button/examples/_floating-icon-lumen-centurylink.vue

```diff
@@ -0,0 +1,62 @@
+<template lang="pug">
+  <ComponentExample titleSize="h4" title="Floating icon buttons" id="floating-icon-lumen-centurylink" :tabs="exampleTabs">
```

Comment:
@soujanreddy doble check all title sizes in the PR as I am finding disconnections with CHi design system doc